### PR TITLE
feat: :sparkles: Tested Tailwind and configured eslint

### PR DESCRIPTION
**Pull request by KiveshVirasamy**
Please check my tailwind

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,69 +1,3 @@
...
-  a:hover {
-    color: #747bff;
-  }
-  button {
-    background-color: #f9f9f9;
-  }
-}
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (sechaba-mache) [inline src/index.css]: This file and the imports for tailwind are not necessary because it is never used. Can be deleted

File: src/App.tsx [hunk 2]
```diff
@@ -1,34 +1,12 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
+/* eslint-disable @typescript-eslint/no-shadow */
```
Comment (sechaba-mache) [inline src/App.tsx]: Add the rule for @typescript-eslint/no-shadow into your eslint.json's rules rather than on that specific file

**Review by sechaba-mache — APPROVED** [review]
Looks good. Just added two minor suggestions on certain files

File: src/index.css (hunk 1 above)
Comment (KiveshVirasamy) [reply src/index.css]: Okay thanks

**Pull request merged**